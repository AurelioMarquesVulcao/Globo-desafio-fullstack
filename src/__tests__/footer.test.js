import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../components/footer';
import toJson from 'enzyme-to-json';


describe('Testing Footer Component', () => {
    it('should render correctly', () => {
        const wrapper = shallow(
            <Footer />
        );
        expect(toJson(wrapper)).toMatchSnapshot();

    });
});