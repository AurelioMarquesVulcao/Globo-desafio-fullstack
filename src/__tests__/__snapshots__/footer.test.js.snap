// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Footer Component should render correctly 1`] = `
<div>
  <div
    className="footer text-white bg-dark"
  >
    <div
      className="nowTv row"
    >
      <img
        alt="description of IMAGEM"
        className="target"
        src={
          Object {
            "ReactComponent": Object {
              "$$typeof": Symbol(react.forward_ref),
              "render": [Function],
            },
            "__esModule": true,
            "default": "target.svg",
          }
        }
      />
      <div
        className="col-md-auto targetOrange"
      >
         
        NO AR
      </div>
      <div />
      <div
        className="col-md-auto"
      >
         Ultima atulização em: 
      </div>
    </div>
    <div
      className="hour"
    />
    <div
      className="nowTime"
    />
  </div>
</div>
`;
